js/src/lib/interfaces/Types.ts

```diff
@@ -657,8 +648,7 @@ export const OTHER_TAGS_SUGGESTIONS = [
 	TAG_NFAA_CONTENT,
 ];
 
-
-export type WidgetInputSample = Record<string | "example_title" | "group", string>;
+export type WidgetInputSample = Record<string | "example_title" | "group", any>;
```

Comment:
The value is sometimes a Record, sometimes an array, etc. I've set `any` but this should be reviewed.

Reply:
I've added 
```
export type WidgetInputSampleValue =
	| string
	| string[]
	| boolean
	| number
	| number[]
	| Record<string, string | string[] | number[]>;
```
that reflects all the input values found in `src/routes/index.svelte` let me know if this sounds good